jest.config.ts

```diff
@@ -7,6 +7,6 @@ module.exports = {
   coveragePathIgnorePatterns: ["node_modules", "mocks"],
   collectCoverage: true,
   coverageReporters: ["json", "lcov", "text", "clover", "json-summary"],
-  reporters: ["default", "jest-junit"],
+  reporters: ["default", "jest-junit", "jest-md-dashboard"],
```

Comment:
I remember there should be something else added to .gitignore from the generated test files not sure if applicable here.

Reply:
Ha yes the generated `.md` file, technically not required but better safe than sorry. Can add it in another PR.